src/main/java/uk/gov/hmcts/ccd/endpoint/std/CallbackEndpoint.java

```diff
@@ -30,6 +32,7 @@ public CallbackEndpoint(final PrintableDocumentListOperation printableDocumentLi
     @ApiResponses(value = {
         @ApiResponse(code = 200, message = "Printable documents list retrieved")
     })
+    @LogAudit(operationType = OperationType.SEARCH_CASE)
```

Comment:
mmm don't understand why this operation is being logged as a search

Reply:
We could probably ignore this endpoint

Reply:
This is a mistake. Will remove and in another PR with v1 will go as 
@LogAudit(operationType = OperationType.VIEW_CASE, caseId = "#caseDetails.reference", caseType = "#caseTypeId",
        jurisdiction = "#jurisdictionId")